rewrite-gradle/src/main/java/org/openrewrite/gradle/util/RewriteStringLiteralValueVisitor.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2022 the original author or authors.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * https://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.openrewrite.gradle.util;
```

Comment:
Wondering if there's a better place to put this. This seems like a pretty common bit of logic that could be a reusable building block.

Reply:
I'm considering refactoring `ChangeDependencyArtifactId`, `ChangeDependencyGroupId`, and `ChangeDependencyVersion` to use this common logic as there are 3 variations of it in each class.

Reply:
Makes sense, let's revisit in a separate issue. Notice in the polished form committed this was turned into a static method rather than visitor, and you can trace the logic to see why. So the question then is does this logic belong as an instance method of `J.Literal`? We've been so far pretty reticent to introduce utility gadgets on AST types to limit API explosion, but maybe this one is really justified.